proto/exocore/oracle/v1/slashing.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package exocore.oracle.v1;
+
+option go_package = "github.com/ExocoreNetwork/exocore/x/oracle/types";
+
+message ValidatorReportInfo {
+  string address = 1;
+  int64 startHeight = 2;
```

buf

Field name "startHeight" should be lower_snake_case, such as "start_height".

(FIELD_LOWER_SNAKE_CASE)